Revision Tests/maria_ptitsyna.md

```diff
@@ -0,0 +1,28 @@
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If only I were smarter, I would have done backup of DB yesterday and only after that applied my PR to code on production. I wish I were as smart as my father.
+
+**_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
+
+> If all students read the theorics information and only after that wrote the translation with analysing sentences, so the progress would be obvious. I regret  everybody wants to learn English by brute force, escapes early stages of education, analysis and skill training to an automatic performance. I wish you don't hurry up. 
+
+**_3. Если бы сегодня мне не нужно было садиться за руль, то вчера бы я знатно набрался с друзьями в ближайшем баре и орал бы в микрофон свою любимую песню. Жаль что автопилот не является повсеместной функцией во всех автомобилях, это могло бы значительно разнообразить досуг водителей._**
+
+> If I didn't need to driving today, I would have excellently drunk with my friends in the nearest bar and cried in a microphone my favourite song yesterday. I wish the autopilot was an all-round feature in all cars, it could much variegate driver's 	leisure. 
+
+**_4. Если бы она проверяла работоспособность билда после слияния своих коммитов с мастером в прошлом, то сейчас бы у нее не было бы общения с клиентом по поводу даунтайма на проекте и финансовых потерях в целом._**
+
+> If she had checked the build performance after merge own commits with master branch in the past, she wouldn't have a conversation with the client about downtime on the project and financial losses in general now.
```

Comment:
build's performance/ the performance of the build
after merging (we use verb+ing after prepositions)
the master branch